docs/tools/offline/mbed-studio.md

```diff
@@ -0,0 +1,193 @@
+
+After you have configured everything, click on the Apply button and then on the Run button.
+
+Watch the primary LED on your connected target begin to blink!
+
+#### Debugging
+
+Now that you have learned the basics of Mbed Studio, it is time to learn how to debug projects within the IDE. There is a simplified debug configuration specifically for easy debugging of Mbed OS projects. There are also additional configurations for more advanced debugging. This section covers the Mbed OS debug configuration, as well as the advanced PyOCD GDB debug configuration.
+
+##### Simple debugging
+
+To begin debugging, click the bug icon, which launches the default debug configuration.
+
+Alternatively, you can create the debug launch configuration and then begin debugging. To create a debug launch configuration, right-click on a project and select `Debug As... > Debug Configurations` from the context menu. (You can also use the menu bar as shown below.)
+
+Once created, the configuration prepopulates all the readily inferred values. Click `Choose...` next to the project textbox, and select the project you wish to debug if it isn’t already populated. After that, connect a board, and the project is ready to debug. The Mbed Debug launch configuration prepopulates the PyOCD GDB server and the GDB client to locations to the internal Mbed Studio tools. Select the Debug button in the lower right portion of the window to start debugging.
+
+Mbed Studio automatically switches into the Mbed Debug perspective when debugging with the Mbed Debug launch configuration. The project also breakes on a default break point. The resulting workspace looks like the image below.
+
+##### Advanced debugging
+
+If you want more control over the options regarding debugging, this section is for you. This section describes how to create a PyOCD GDB debug configuration. This configuration allows you to change more settings, such as ports and executable locations for the GDB client and server.
+
+To begin debugging, create the debug launch configuration and set some variables. (You may use your own values if you don’t want to use the default provided here.) To create a debug launch configuration, as before, right-click on a project, and select `Debug As... > Debug Configurations` from the context menu.
+
+A window for the debug configurations appears. Select the `GDB PyOCD Debugging` category, and create a new launch configuration. 
+
+You can click the new icon (a page with a `+` in the upper right corner), or right-click on the category and select `New`. See the screenshot above on the right.
+
+Next, configure the binary you want to use for debugging. This is the `.elf` file located within the project directory and inside `BUILD/<target>/GCC_ARM`. (For this example, it is `BUILD/K64F/GCC_ARM`.) Currently, this is configured manually, so switching to a different target to run the same project requires you to update this field in the debug configuration. See the below left screenshot.
```

Comment:
Note: There is no image below left. Please add the image to the "images" folder at the top level of these docs, and include the AWS link in this .md file.